blazor/datagrid/filter-menu.md

```diff
@@ -223,4 +223,105 @@ The following screenshot represents CheckBox filter
 
 ### Limitations of using different filter types in different columns
 
-The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you cannot use these filter types along with filterBar type (default filter type). Because the filterbar type requires UI level changes with other filter types. For all other filter types, icons will be rendered in the column header.
\ No newline at end of file
+The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you cannot use these filter types along with filterBar type (default filter type). Because the filterbar type requires UI level changes with other filter types. For all other filter types, icons will be rendered in the column header.
+
+## Filter by multiple keywords using filter menu
+
+By default, the filtering action is performed based on the single keyword filter value from the built-in component of the filter menu dialog. Now the DataGrid has an option to perform filtering actions based on multiple keywords instead of a single keyword alone. This can be achieved by using the [FilterTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_FilterTemplate) feature of the Grid.
```

Comment:
this can be achieved using FilterTempalte and FilterBYColumnAsync method of Grid.